src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -18,4 +18,11 @@
      * @return an user with its information.
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.